velox/functions/sparksql/Register.cpp

```diff
@@ -158,7 +158,7 @@ void registerFunctions(const std::string& prefix) {
 
   registerRegexpReplace(prefix);
 
-  registerFunction<JsonExtractScalarFunction, Varchar, Varchar, Varchar>(
```

Comment:
@mbasmanova, I think we can just remove this line and the above header including.

Currently, Gluten is using another implementation in a downstream velox. And we have a pr for upstream: see https://github.com/facebookincubator/velox/pull/5179.

Reply:
@PHILO-HE Got it. Thank you for pointing this out. I'll remove this.